main.go

```diff
@@ -38,8 +38,15 @@ func main() {
 
 	//This method takes in the URL path "/" and a function that takes in a response writer, and a http request.
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		d := LoadData()
+		d, err := LoadData()
+
+		if err != nil {
+			fmt.Println("Error loading data")
```

Comment:
Probably send the caller a real error here:
```
w.WriteHeader(http.StatusInternalServerError)
w.Write([]byte(`{"message": "error loading data"}`))
return
```

Reply:
maybe something similar to this : 
line68 `http.Error(w, err.Error(), http.StatusInternalServerError)`